.github/workflows/daily-buildpulse.yml

```diff
@@ -3,8 +3,8 @@ name: Daily BuildPulse run
 on:
   workflow_dispatch:
   schedule:
-    # https://crontab.guru - “At every 30th minute past every hour from 3 through 6.”
-    - cron: '*/30 3-6 * * *'
+    # https://crontab.guru - “At every 30th minute past every hour from 3 through 4.”
```

Comment:
That means just 2 builds? I am not quite sure that would be enough for flakiness detection.

Reply:
We only need 2 runs on the same commit for Buildpulse technically, but this would get for 4 runs, actually.
The other problem is that we are not doing anything with the reports currently anyway?

Before = 8 runs (UTC time)
- 3:00
- 3:30
- 4:00
- 4:30
- 5:00
- 5:30
- 6:00
- 6:30

After = 4 runs
- 3:00
- 3:30
- 4:00
- 4:30